src/utils/request.ts

```diff
@@ -0,0 +1,16 @@
+import { parse as parseUrl } from 'url';
+import parseCurl from 'parse-curl';
+
+import { Request, Method } from '../domain/entity';
+
+export function createRequestFromCurl(curlCommand: string) {
+  const parsedCommand = parseCurl(curlCommand);
+  const url = parseUrl(parsedCommand.url);
+
+  return new Request(
+    parsedCommand.method as Method,
+    url.path || '',
```

Comment:
This implementation breaks when Memento `targetUrl` has some path parts in it (like `http://pokeapi.co/api/v2`), because this output path will not be correct for reading the request response.